### DISCUSSION TITLE
Being a member of the repository

### DISCUSSION THREAD

**Discussion by ycollet** [audinux/fedora-spec#37] [General]
Hello @jn64

I was wondering if you wanted to become a member of this project ?
Tell me if you're interested, I will add you.

Best regards,

Yann

**Comment by jn64** [comment]
Hi Yann, thank you for the invitation! I would be happy to join the project.

Actually, one of the reasons I became interested in packaging in the past few years is because of audinux/linuxmao :) I hope I can be of help, even though I still have a lot to learn.

**Comment by ycollet** [comment]
I am still learning after all these years :)
My principal source of knowledge: https://src.fedoraproject.org/
I will add you

**Reply by ycollet** [reply]
Done normally. I hope I have done things right :)

**Reply by jn64** [reply]
Do you know how I can use the [Projects](https://github.com/audinux/fedora-spec/projects) in this repo? It seems to be limited to "audinux" organisation.

I can also do it on my own account, so no problem if it's not possible due to GitHub things.

---

Screenshot:
![Screenshot 2023-06-12 at 11-31-54 Build software better together](https://github.com/audinux/fedora-spec/assets/23169302/df0aae24-9761-40f6-8d2b-d02e1dce0d1a)

**Reply by ycollet** [reply]
I will check tonight. I have maybe something to do on my side to enable this.

**Reply by jn64** [reply]
I can use it after joining the org. Thanks!

**Comment by ycollet** [comment]
I was wondering if having a discord channel was a good idea for discussion ?

**Comment by ycollet** [comment]
I just created something on discord:
https://discord.gg/Tgdard5V

**Reply by jn64** [reply]
Discord rejected my domain name many years ago :) I don't mind creating an account (if they let me)... I'll try now

Or do you use matrix?

**Comment by ycollet** [comment]
I never tried matrix. I will check tonight to create a server on matrix.

**Reply by jn64** [reply]
Yup, I am still stuck on verification, sorry. Well... I can make a temporary gmail or something if it's easier to use discord

If you want to try matrix, you can use Fedora's web client https://chat.fedoraproject.org

You can use FAS to login, it will give you `@username:fedora.im`. `fedora.im` is the Fedora "homeserver"

I think we can create our own public/private room (aka channel) on Fedora homeserver, with a name like `#audinux:fedora.im`.

**Reply by ycollet** [reply]
A priori it's OK now: #audinux.fedora.im should work
